packages/prosess-vilkar-overstyring/src/components/VilkarresultatMedOverstyringForm.tsx

```diff
@@ -0,0 +1,353 @@
+  aksjonspunktCodes: string[];
+  aksjonspunkter: Aksjonspunkt[];
+  avslagsarsaker: Kodeverk[];
+  behandlingsresultat: {
+    type: Kodeverk;
+  };
+  behandlingId: number;
+  behandlingVersjon: number;
+  behandlingType: Kodeverk;
+  customVilkarIkkeOppfyltText?: {
+    id: string;
+    values?: any;
+  };
+  customVilkarOppfyltText?: {
+    id: string;
+    values?: any;
+  };
+  erMedlemskapsPanel: boolean;
+  erOverstyrt?: boolean;
+  erVilkarOk?: boolean;
+  hasAksjonspunkt: boolean;
+  isReadOnly: boolean;
+  kanOverstyreAccess?: {
+    isEnabled: boolean;
+  };
+  lovReferanse?: string;
+  medlemskapFom: string;
+  originalErVilkarOk?: boolean;
+  overrideReadOnly: boolean;
+  overstyringApKode: string;
+  panelTittelKode: string;
+  periodeFom: string;
+  periodeTom: string;
+  status: string;
+  submitCallback: (props: SubmitCallback[]) => void;
+  toggleOverstyring: (overstyrtPanel: SetStateAction<string[]>) => void;
+}
+
+interface StateProps {
+  isSolvable: boolean;
+}
+
+/**
+ * VilkarresultatForm
+ *
+ * Presentasjonskomponent. Viser resultat av vilkårskjøring når det ikke finnes tilknyttede aksjonspunkter.
+ * Resultatet kan overstyres av Nav-ansatt med overstyr-rettighet.
+ */
+export const VilkarresultatMedOverstyringForm = ({
+  panelTittelKode,
+  erOverstyrt,
+  isReadOnly,
+  overstyringApKode,
+  lovReferanse,
+  isSolvable,
+  erVilkarOk,
+  originalErVilkarOk,
+  customVilkarIkkeOppfyltText,
+  customVilkarOppfyltText,
+  erMedlemskapsPanel,
+  hasAksjonspunkt,
+  avslagsarsaker,
+  overrideReadOnly,
+  kanOverstyreAccess,
+  aksjonspunktCodes,
+  toggleOverstyring,
+  reset,
+  handleSubmit,
+  submitting,
+  pristine,
+  periodeFom,
+  periodeTom,
+}: VilkarresultatMedOverstyringFormProps & StateProps & InjectedFormProps) => {
+  const togglePa = () => {
+    toggleOverstyring(oldArray => [...oldArray, overstyringApKode]);
+  };
+  const toggleAv = () => {
+    reset();
+    toggleOverstyring(oldArray => oldArray.filter(code => code !== overstyringApKode));
+  };
+
+  useEffect(() => {
+    return () => {
+      reset();
+    };
+  }, [periodeFom, periodeTom]);
+
+  return (
+    <form onSubmit={handleSubmit}>
+      <FlexContainer>
+        <FlexRow>
+          {!erOverstyrt && originalErVilkarOk !== undefined && (
+            <FlexColumn>
+              <Image className={styles.status} src={originalErVilkarOk ? innvilgetImage : avslattImage} />
+            </FlexColumn>
+          )}
+          <FlexColumn>
+            <Undertittel>
+              <FormattedMessage id={panelTittelKode} />
+            </Undertittel>
+          </FlexColumn>
+          {lovReferanse && (
+            <FlexColumn>
+              <EtikettLiten className={styles.vilkar}>{lovReferanse}</EtikettLiten>
+            </FlexColumn>
+          )}
+        </FlexRow>
+        <FlexRow>
+          <FlexColumn>
+            {originalErVilkarOk && (
+              <>
+                <VerticalSpacer eightPx />
```

Comment:
Lag ny komponent her, dette gjentar seg:
`VerticalSpacer - Element - FormattedMessage`

Spacingen trenger ikke å bli med i komponenten, men kan legges over slik at det gjelder likt for alle. Må forsåvidt ikke være en ny komponent heller, bare sørg for at vi ikke repeterer oss selv for mye her, ettersom det eneste som endrer seg er verdiene på conditions for rendering og en tekstnøkkel.